### ISSUE TITLE
https://www.dhakatribune.com/fifa-world-cup-2022/2022/11/28/boxer-alvarez-threatens-messi-over-world-cup-jersey-insult

### ISSUE THREAD

**Issue by Bhooty** [AdguardTeam/AdguardFilters#136034]
### Prerequisites

- [X] This site DOES NOT contains sexually explicit material, otherwise use [NSFW-specific form](./new?template=bug_report_NSFW.yml);
- [X] Filters were updated before reproducing an issue;
- [X] AdGuard product version is up-to-date;
- [X] Browser version is up-to-date;
- [X] If the site or app is broken, disabling AdGuard protection resolves an issue.

### What product do you use?

AdGuard for iOS

### AdGuard version

4.4.4

### What type of problem have you encountered?

Missed ads or ad leftovers

### Which browser(s) do you use?

Safari

### Which device do you use?

Mobile

### Where is the problem encountered?

https://www.dhakatribune.com/fifa-world-cup-2022/2022/11/28/boxer-alvarez-threatens-messi-over-world-cup-jersey-insult

### What filters do you have enabled?

AdGuard Base filter, AdGuard Mobile Ads filter, AdGuard Tracking Protection filter, AdGuard Social Media filter, AdGuard Annoyances filter

### Add your comment and screenshots

There are two add leftovers at the end of each articles

and in www.bangla.bdnews24.com

there is a ad leftover at the top

![D0C8FD0B-851F-4B6E-9EB5-6351E95629F9](https://user-images.githubusercontent.com/46019727/204341145-1e94c77f-aadb-46f3-a0a9-0498b103e8e9.png)

![B87738AD-35F9-4BA0-BB83-2F549B9B6EE9](https://user-images.githubusercontent.com/46019727/204340885-ead3fae9-f385-455c-a7c4-f4738436c6f0.png)

### Privacy

- [X] I agree to follow this condition

**Comment by Bhooty** [comment]
@jellizaveta my bad sir
I gave the old link by mistake
Should i open a new issue or give the link here?

**Comment by jellizaveta** [comment]
@Bhooty No, it's okay. I have added rules for `bangla.bdnews24.com` , filter update should be available in 1-2 hours.
